app/main.py

```diff
@@ -194,7 +190,8 @@ async def to_approve():
     return needs_approval
 
 
-@repeat_every(seconds=60 * 60)
+@app.on_event("startup")
+@repeat_every(seconds=60 * 4)
 @app.get("/advance-all/")
```

Comment:
This seems to be an issue with Fast api and it does not recognize a timer without a start up event. This should fix the bug.